api/pom.xml

```diff
@@ -23,8 +23,8 @@
         <relativePath>../</relativePath>
     </parent>
 
-    <artifactId>microprofile-health-api</artifactId>
-    <name>MicroProfile Health API</name>
+    <artifactId>microprofile-health-check-api</artifactId>
```

Comment:
I wouldn't be a fan. Consistency is good, but not breaking users is better.

Reply:
Yes, that's the reason why I asked on the ML. I'll probably revert that